mlflow/ml-package-versions.yml

```diff
@@ -577,6 +577,27 @@ langchain:
         ]
     run: |
       pytest tests/langchain/test_langchain_model_export.py
+  autologging:
+    minimum: "0.1.0"
+    maximum: "0.1.0"
+    requirements:
+      ">= 0.0.0":
+        [
+          "pyspark",
+          "transformers",
+          "tensorflow",
+          "openai",
+          "google-search-results",
+          "psutil",
+          "faiss-cpu",
+          "langchain-experimental",
+          "numexpr",
+          "spacy",
+        ]
+    run: |
+      # For MlflowCallback text analysis
+      python -m spacy download en_core_web_sm
```

Comment:
where is dataset used?

Reply:
It's used inside MLflowCallbackHandler's analyze_text function https://github.com/langchain-ai/langchain/blob/7d444724d7582386de347fb928619c2243bd0e55/libs/community/langchain_community/callbacks/mlflow_callback.py#L76

Reply:
https://github.com/langchain-ai/langchain/blob/7d444724d7582386de347fb928619c2243bd0e55/libs/community/langchain_community/callbacks/mlflow_callback.py#L272

Reply:
Actually I think I should add a warning at that line to ask user to run this command if `spacy.load("en_core_web_sm")` fails, and wrap it into a try catch

Reply:
I see, so MLflowCallbackHandler depends on the `en_core_web_sm` dataset existing?

Reply:
Yes